marketpulse/geo/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+
+class LocaltionBase(models.Model):
+    """Model for geolocation data."""
+
+    country = models.CharField(max_length=120, unique=True)
+    country_code = models.CharField(max_length=2)
```

Comment:
We can use `product_details` to get the country codes for this - if this something that we want to keep.